test/lit/wasm-split/multi-memory-lowering-export.wast

```diff
@@ -0,0 +1,9 @@
+;; RUN: wasm-opt %s --enable-multi-memories --multi-memory-lowering --enable-bulk-memory --enable-extended-const --enable-simd --enable-threads -S -o - | filecheck %s
```

Comment:
For the non-error tests, it would be good to put the magic comment at top so they can be auto-updated.